tests/modules/aclcheck.c

```diff
@@ -157,13 +160,13 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     if (RedisModule_CreateCommand(ctx,"set.aclcheck.key", set_aclcheck_key,"",0,0,0) == REDISMODULE_ERR)
         return REDISMODULE_ERR;
 
-    if (RedisModule_CreateCommand(ctx,"test.aclcheck.key.user.deleted", test_aclcheck_key_user_deleted,"",0,0,0) == REDISMODULE_ERR)
+    if (RedisModule_CreateCommand(ctx,"publish.aclcheck.channel", publish_aclcheck_channel,"",0,0,0) == REDISMODULE_ERR)
```

Comment:
the convention is that the module name comes first.
i think these should be `aclcheck.xxxxx`